data/src/generate_tsv_summary.py

```diff
@@ -97,17 +124,29 @@ def main() -> None:
     with open(README_PATH, "w", encoding="utf-8") as sink:
         print(
             "| Link | ISO 639-2 Code | ISO 639 Language Name "
-            "| Wiktionary Language Name | Case-folding "
-            "| Phonetic/Phonemic | # of entries |",
+            "| Wiktionary Language Name | Script | Dialect | Filtered "
+            "| Phonetic/Phonemic | Case-folding | # of entries |",
             file=sink,
         )
         print(
-            "| :---- | :----: | :----: | :----: | :----: | :----: | ----: |",
+            "| :---- |" + "".join([" :----: |" * 8]) + " ----: |",
```

Comment:
Sorry for late-breaking observation but I just noticed this. What you're doing here, I think, is conceptually:

```
"".join([str])
```

You can just write:

```str```

instead. That would look like:

````"| :---- |" + " :----: |" * 8 + " ----: |",```

I think.